src/tree/containers/ContainersTreeItem.ts

```diff
@@ -82,6 +82,15 @@ export class ContainersTreeItem extends LocalRootTreeItemBase<DockerContainerInf
                 return ti1.label === NonComposeGroupName ? 1 : -1;
             }
         }
+        if (this.groupBySetting === 'Label'
```

Comment:
I'd turn this into an `else if (...)` grouped with the `if` above, and maybe put in an extra newline before and after the block that should've always been there.

Reply:
Semantically it's the same, but looks a little neater IMO.